commands.txt

```diff
@@ -0,0 +1,176 @@
+## Commands
+This section lists command(s) run by WORKFLOW workflow
+
+* Running WORKFLOW
+
+=== Description here ===.
+
+<<<
```

Comment:
This needs to be pretty-fied (see **delly** or **gridss** for reference). Note that we use ``` in README files for code blocks. Also, it helps to edit the body of a command block and make it look more readable (although in the case of **MAVIS** it may be a little challenging since it uses a large embedded python script)